packages/create-remix/templates/fly-stack/.github/workflows/deploy.production.yml

```diff
@@ -0,0 +1,26 @@
+name: Fly Deploy Production
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
```

Comment:
```suggestion
        uses: styfle/cancel-workflow-action@0.9.1
```

Reply:
haha, yeah forgot that isn't needed anymore 😅, thanks!